threadedServer.py

```diff
@@ -28,12 +28,25 @@
 # Doesn't really matter what's in here, just make sure only 2 for now
 # Otherwise have to change the logic of the server
 player_data = [
-    Player(0, 0, 50, 50, "Red", 100),
-    Player(50, 50, 60, 60, "Green", 95)
+    "Hello server data",        # P1 data to be sent to P2
+    "This from the Phone"       # Changed immediately upon connection from P2
 ]
-
+current_player = 0      # The current player
+players = []        # To allow for reconnections
```

Comment:
This will be useful later in future. Not yet implemented. Also moved current_player up here to also be used in the 'threaded_client' on event of disconnect